fiftyone/utils/labelbox.py

```diff
@@ -620,10 +620,13 @@ def upload_samples(self, samples, anno_key, backend):
         project_name = config.project_name
         members = config.members
         classes_as_attrs = config.classes_as_attrs
-        is_video = (samples.media_type == fomm.VIDEO) or (
```

Comment:
@ehofesmann Checking for grouped datasets at *upload time* is not necessary since the Labelbox backend's `supported_media_types` property already forces the user to pass in a `select_group_slices()` view when annotating video datasets.

This was acknowledged in https://github.com/voxel51/fiftyone/pull/4073 but the extra code was still added.

Reply:
Ah yup, you're right. I must have misunderstood what ben was saying, but this should just be checked when downloading samples